### ISSUE TITLE
Raise exceptions if file is invalid

### ISSUE THREAD

**Issue by gianfranco-s** [tomaspf9/ReadMyDuck#25]
Files that
- have no text
- are too long

Should raise exceptions

**Comment by gianfranco-s** [comment]
@NCellini @tomaspf9 please provide evidence of the errors you found here

**Comment by gianfranco-s** [comment]
@NCellini has your PR #30 solved the part for files too long?

**Comment by NCellini** [comment]
Yes, the PR #30 solved that part. it was a problem with the cookies and with the change to filesystem is solved now.
